lib/src/box.dart

```diff
@@ -257,6 +257,10 @@ class Box<T> {
     }
   }
 
+  /// When no conditions are required on properties
+  QueryBuilder get emptyQuery =>
+      QueryBuilder<T>(_store, _fbManager, _modelEntity.id.id);
+
```

Comment:
Not sure we need a new method/getter. In some other language bindings it's just box.query() with no arguments.

Reply:
Square brackets would do the job.

```dart
QueryBuilder get query(..., [Condition c]) =>
```

Feel free to make the modification.